pkg/apis/serving/v1beta1/configmap.go

```diff
@@ -110,8 +110,10 @@ type InferenceServicesConfig struct {
 
 // +kubebuilder:object:generate=false
 type IngressConfig struct {
-	IngressGateway     string `json:"ingressGateway,omitempty"`
-	IngressServiceName string `json:"ingressService,omitempty"`
+	IngressGateway          string `json:"ingressGateway,omitempty"`
+	IngressServiceName      string `json:"ingressService,omitempty"`
+	LocalGateway            string `json:"localGateway,omitempty"`
+	LocalGatewayServiceName string `json:"localGatewayService,omitempty"`
```

Comment:
@yuzisun, I've missed that, but it generates a name mismatch violation in the `make generate`. We can either match the names or add it in the violation exceptions.